tests/Behat/Context/Application/CreditMemoContext.php

```diff
@@ -108,6 +109,32 @@ public function creditMemoTotalShouldBe(int $total): void
         Assert::same($this->creditMemo->getTotal(), $total);
     }
 
+    /**
+     * @Then /^its subtotal should be ("[^"]+")$/
```

Comment:
```suggestion
     * @Then its subtotal should be :subtotal
```
is regexp really needed?

Reply:
Yup, because I need a value in integer here